sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -100,6 +100,17 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase {
       }
     }
   }
+
+  test("no database specified") {
+    Seq(
+      s"SHOW TABLES IN $catalog",
+      s"SHOW TABLE EXTENDED IN $catalog LIKE '*tbl'").foreach { showTableCmd =>
+      val errMsg = intercept[AnalysisException] {
+        sql(showTableCmd)
+      }.getMessage
+      assert(errMsg.contains("multi-part identifier cannot be empty"))
```

Comment:
Can we make the error message more explicit? `Namespaces in V1 catalog can only have a single name part.`

Reply:
Since it may affect many tests, let's update the error message in a new PR.

Reply:
I would prefer to make such changes separately if you don't mind because such change will cause modification that are unrelated to this PR.

Reply:
Here is the PR: https://github.com/apache/spark/pull/30947 . I have fixed an obvious place. Let's see where Jenkins build will fail.